config/dev.webpack.config.js

```diff
@@ -69,7 +69,15 @@ plugins.push(
             './DetailWrapper': resolve(__dirname, '../src/modules/DetailWrapper.js'),
             './OsFilterHelpers': resolve(__dirname, '../src/modules/OsFilterHelpers.js'),
             './systemProfileStore': resolve(__dirname, '../src/store/systemProfileStore.js')
-        }
+        },
+        shared: [
```

Comment:
This module should be shared automatically if you have your shared webpack config up to date.

Reply:
Just wanted to be double-sure it's shared. But yeah, let's just leave it on the shared webpack config. Fixed.